### DISCUSSION TITLE
Welcome to badaso Discussions!

### DISCUSSION THREAD

**Discussion by rizkiheryandi** [uasoft-indonesia/badaso#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by febrihidayan** [comment]
Apakah sudah menerapkan Agile dan Scrum untuk mengembangkan Badaso?

**Reply by rizkiheryandi** [reply]
Sudah pak 😊

**Reply by febrihidayan** [reply]
Cara berkonstribusi seperti apa kalau di terapkan di Laravel?
Apakah harus membuat direktori Packages di direktori utama Laravel

**Reply by rizkiheryandi** [reply]
Please read [this ](https://github.com/uasoft-indonesia/badaso/blob/main/CONTRIBUTING.md)for contributing

Thank you 😊
